### PR TITLE
Upgrade to OpenLayers v6.x

### PR DESCRIPTION
**Pull request by annarieger**
### BREAKING CHANGE

Upgrades `ol-util` package to be able to work with `ol` v 6.x and higher.

Additionally updated autogenerated API documentation.

Please review @terrestris/devs

### CODE REVIEW

File: src/FileUtil/FileUtil.spec.js [hunk 1]
```diff
@@ -48,10 +48,6 @@ describe('FileUtil', () => {
       map = TestUtil.createMap();
     });
 
-    afterEach(() => {
-      TestUtil.removeMap(map);
-    });
```
Comment (dnlkoch) [inline src/FileUtil/FileUtil.spec.js]: Why is this needed?

File: README.md [hunk 2]
```diff
@@ -732,6 +777,41 @@ mappings to allow automatic CRS detection by `OpenLayers` here.
...
+Adds leading zero to all values less than 10 and returns this new
+zerofilled value as String. Values which are greater than 10 are not
+affected.
+
+##### Parameters
+
+-   `value` **[number](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Number)** Value to be zerofilled.
+
+Returns **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** converted value with leading zero if necessary.
+
```
Comment (dnlkoch) [inline README.md]: Docs are available [here](https://terrestris.github.io/ol-util/latest/index.html) and will be updated automatically. I suggest to delete the docs from the README completely (in a follow up MR).

File: src/FileUtil/FileUtil.spec.js (hunk 1 above)
Comment (annarieger) [reply src/FileUtil/FileUtil.spec.js]: Map removal leads to the following error, cause [`renderer`](https://github.com/openlayers/openlayers/blob/master/src/ol/PluggableMap.js#L1292) seems to be null at the moment where the test tries to access the map then:
```
TypeError: Cannot read property 'renderFrame' of null

at Map.PluggableMap.renderFrame_ (node_modules/ol/src/PluggableMap.js:1292:20)
      at Map.<anonymous> (node_modules/ol/src/PluggableMap.js:198:12)
      at Timeout.callback [as _onTimeout] (node_modules/jsdom/lib/jsdom/browser/Window.js:678:19)
```

The error is possibly due to https://github.com/openlayers/openlayers/pull/9637

For now, I've commented the `afterEach` test block and  added a `TODO` to investigate it later.

I've also added the corresponding issue.

File: README.md (hunk 2 above)
Comment (annarieger) [reply README.md]: Ok, will be done in a separate MR

Comment (annarieger) [reply README.md]: See #151
